tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/tests/scoping/ComposedMemberCachingTest.xtend

```diff
@@ -150,19 +152,39 @@ class ComposedMemberCachingTest extends AbstractN4JSTest {
 		scope.getAllElements();
 		assertTrue("scoping should not have added anything to composed member cache",
 			script.module.composedMemberCaches.empty);
-		
+
 		// only for completeness, we test the counter example using ordinary member scoping:
 		val scope2 = memberScopingHelper.createMemberScope(receiverTypeRef, propAccess, false, true);
 		scope2.getAllElements();
 		assertTrue("scoping should have added several members to composed member cache",
 			script.module.composedMemberCaches.head.cachedComposedMembers.size>1); // at time of writing, 16 members were added (but exact count might change over time)
 	}
 
+	def private Script parseAndPostProcessAndDeleteCachedComposedMembers(CharSequence code) {
+		val script = code.parseAndValidateSuccessfully;
+		assertTrue((script.eResource as N4JSResource).isFullyProcessed); // ensure validation did trigger post-processing
+		// While resolving the proxies during validation, composed members will have been added to the composed member
```

Comment:
It took me a while to understand this comment. Probably it is better to say something like "Here, we are are going to call scoping manually and test that composed member cache is created correctly"